### PR TITLE
FEATURE: CL-7120 Webpack

### PR DESCRIPTION
**Pull request by akloboucnik**

### CODE REVIEW

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/GD-JS-Pull-Request/54/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/GD-JS-Pull-Request/55/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/GD-JS-Pull-Request/56/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/GD-JS-Pull-Request/57/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/GD-JS-Pull-Request/58/

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2007-2015, GoodData(R) Corporation. All rights reserved.
```
Comment (jankopriva) [inline webpack.config.js]: can't u just merge this into grunt's config definition or, better get the grunt's config and put the stuff in here...?

Comment (akloboucnik) [inline webpack.config.js]: I've taken it from some webpack grunt integration example and I kinda like it - the common config for webpack is separated and the differences for each grunt subtask are overridden in Gruntfile.js

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/GD-JS-Pull-Request/59/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/GD-JS-Pull-Request/60/

**Pull request merged**